tests/data/test_db.sql

```diff
@@ -2217,22 +2217,22 @@ UNLOCK TABLES;
 DROP TABLE IF EXISTS `location`;
 
 CREATE TABLE `location` (
-  `id` int(11) NOT NULL AUTO_INCREMENT,
-  `language` varchar(5) COLLATE utf8mb4_unicode_ci NOT NULL,
-  `extra_id` int(11) NOT NULL,
-  `title` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
-  `street` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
-  `number` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
-  `zip` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
-  `city` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
-  `country` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
-  `lat` float DEFAULT NULL,
-  `lng` float DEFAULT NULL,
-  `show_overview` tinyint(1) NOT NULL DEFAULT '1',
-  `created_on` datetime NOT NULL,
-  `edited_on` datetime NOT NULL,
-  PRIMARY KEY (`id`)
-) ENGINE=MyISAM DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+	`id` int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
why tabs?

Reply:
well it's a bit all over the place in the entire file so... you want me to change them all? I'd be glad to

Reply:
Done in 88acd04e9